test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/TokenMovement.java

```diff
@@ -140,22 +165,48 @@ public TokenTransferList specializedFor(HapiApiSpec spec) {
 		return scopedTransfers.build();
 	}
 
+	public TokenTransferList specializedForNft(HapiApiSpec spec) {
+		var scopedTransfers = TokenTransferList.newBuilder();
+		var id = isTrulyToken() ? asTokenId(token, spec) : HBAR_SENTINEL_TOKEN_ID;
+		scopedTransfers.setToken(id);
+		if (sender.isPresent() && receiver.isPresent()) {
+			scopedTransfers.addNftTransfers(adjustment(sender.get(), receiver.get(), serialNum, spec));
+		}
+
+		return scopedTransfers.build();
+	}
+
 	private AccountAmount adjustment(String name, long value, HapiApiSpec spec) {
 		return AccountAmount.newBuilder()
 				.setAccountID(asId(name, spec))
 				.setAmount(value)
 				.build();
 	}
 
+	private NftTransfer adjustment(String senderName, String receiverName, long value, HapiApiSpec spec) {
+		return NftTransfer.newBuilder()
+				.setSenderAccountID(asId(senderName, spec))
+				.setReceiverAccountID(asId(receiverName, spec))
+				.setSerialNumber(value)
+				.build();
+	}
+
 	public static class Builder {
 		private final long amount;
+		private long serialNum;
 		private final String token;
 
 		public Builder(long amount, String token) {
 			this.token = token;
 			this.amount = amount;
 		}
 
+		public Builder(long amount, long serialNum, String token) {
```

Comment:
We do not need `amount` - its always `1`

Reply:
We do need since we already have Builder(long, String) constructor and serialNum is long